pkg/authenticator/config/config_test.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"fmt"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/log"
+	"os"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+var mockSuccessfulOptions = Options{
+	successfulMockReadSSLCertFunc,
+	 successfulMockNewUserNameFunc,
+}
+
+var environmentValues = map[string]string{
+	"CONJUR_SSL_CERTIFICATE": "testSSLCert",
+	"CONJUR_CERT_FILE": "testSSLCertFile.txt",
+	"CONJUR_AUTHN_URL": "filepath",
+	"CONJUR_ACCOUNT": "testAccount",
+	"CONJUR_AUTHN_LOGIN": "host",
+	"MY_POD_NAMESPACE": "testNameSpace",
+	"MY_POD_NAME": "testPodName",
+	"CONJUR_VERSION": "",
```

Comment:
nit: Would be good to sort these since it's hard to find a variable in an unordered list